src/averageAges.js

```diff
@@ -1,61 +1,61 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const manList = people.filter(
+    century
+      ? person => {
+        const centuryValid = Math.ceil(person.died / 100) === century;
+
+        return centuryValid && person.sex === 'm';
+      }
+      : person => person.sex === 'm');
```

Comment:
Why not use if() {}?

Reply:
@solaryasha @Andr1yK  if I could use it, it wasn't be so painfull for me ))